amqp/src/main/mima-filters/1.1.x.backwards.excludes

```diff
@@ -0,0 +1,3 @@
+# PR #1921
+# new argument to private constructor
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.stream.alpakka.amqp.AmqpWriteSettings.this")
```

Comment:
As we've now released 2.0.0-M1, you need to rename this file to `2.0.0-M1.backwards.excludes`.

Reply:
Done.